.github/workflows/algolia-upload-index.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
We had a mix of `ubuntu-latest` and `ubuntu-20.04` in our yamls. I switched to using `ubuntu-20.04` everywhere to avoid possible breaking changes when `ubuntu-22.04` becomes the default.